server.js

```diff
@@ -35,6 +35,10 @@ app.use(jsPath, express.static('./dist/client'));
 app.engine('hbs', exphbs({extname: '.hbs'}));
 app.set('view engine', 'hbs');
 
+const defaultPluginTitle = 'Jenkins Plugins';
+const defaultPluginDescription = 'Jenkins – an open source automation server which enables developers around the world to reliably build, test, and deploy their software';
```

Comment:
I suggest changing the variable to indicate the name and description are not specific to the plugin site

```suggestion
const defaultPluginSiteDescription = 'Jenkins – an open source automation server which enables developers around the world to reliably build, test, and deploy their software';
```